docs/bot/setup.md

```diff
@@ -1,7 +1,13 @@
 # Setting up Draupnir
 
-It is recommended to use [Pantalaimon](https://github.com/matrix-org/pantalaimon) so your management
-room can be encrypted. This also applies if you are looking to moderate an encrypted room.
+While in the past there has been a recomendation to use [Pantalaimon](https://github.com/matrix-org/pantalaimon) to support
+encrypted management rooms or use of the experimental Rust Crypto for the same effect this recomendation is no longer in effect.
```

Comment:
```suggestion
## Encrypting the management room

We do not recommend setting up an encrypted management room, and we cannot provide support for setting one up. This is because Pantalaimon has consistently confused beginners that are setting it up with Draupnir, and we do not use it ourselves. Experienced system admins are welcome to attempt to set up Pantalaimon, but must be aware that any issues that they experience getting Draupnir to start are extremely likely to be from the Pantalaimon configuration and not Draupnir itself. 
```

We need to be very explicit Cat, we can't just make a half assed recommendation because they will ignore it.